int8[6] pos_ctrl
int8[6] speed_ctrl
int8[6] current_ctrl


int8[6] pos_fb
int8[6] speed_fb
int8[6] current_fb

uint8 which_choose
uint8 choose_pos     = 1
uint8 choose_speed   = 2
uint8 choose_current = 3
uint8 choose_none    = 4

uint8 shutdown